app/release_metadata.json

```
@@ -1,11 +1,11 @@
 {
-  "version": "7.0.0-beta",
-  "stability": "beta",
+  "version": "7.0.0-rc",
+  "stability": "rc",
   "minimum_php_version": "8.2.0",
   "maximum_php_version": "8.4.99",
   "minimum_mysql_version": "8.4.0",
   "minimum_mariadb_version": "10.11.0",
   "show_php_version_warning_if_under": "8.2.0",
   "minimum_mautic_version": "5.0.0",
```

```suggestion
  "minimum_mautic_version": "6.0.0",
```
Keep one major version maximum. In the release build, vendor files are compared only between the two specified versions; versions in between are ignored, which can cause upgrade issues.

---

Good catch @matbcvo! However this will mean people can't upgrade from 5 to 7, right? It will not show someone on Mautic 5.2 the option to update directly to 7, they'll have to go via 6 first, correct?

I don't know how annoying that might be for the large number of users who are going to stick with 5.2 LTS and then update directly to 7, for example.

---

Yes, when I fixed the release build, I skipped adding support for comparing versions between the two specified ones. It was a quick solution to get the release out. Also we've always kept only one major version in the past.

---

But we want to be able to upgrade from M5. As M6 was just a short term version and many M5 users are waiting for LTS (M7) to upgrade.

---

[As there were no issues upgrading from 5.0.0 to 7.0.0-beta](https://github.com/mautic/mautic/actions/runs/18652003582/job/53172313852), you can skip my suggestion and proceed with merging the PR. I just prefer to avoid any potential upgrade issues with vendor files.

---

Ok. Let's get the RC out and test all the upgrade scenarios we can think of with it. We can release RC2 if we find some issue.